### ISSUE TITLE
Ability to clear a single mail message

### ISSUE THREAD

**Issue by JasonHorkles** [EssentialsX/Essentials#3639]
### Feature request

**Feature description**
Doing the command `/mail delete <mail number>` would be useful if you want to delete just one message instead of clearing all your mail. This way you can have a cleaner inbox but keep the things you want to keep for a bit longer.

**How the feature is useful**
You won't have to go into the user's userdata file just do clear up some mail and they can do it themselves.

**Comment by Chew** [comment]
Could also be /mail read to "mark it as read" but that might be something else or get confused

**Comment by triagonal** [comment]
Duplicate of #3097.

**Comment by JasonHorkles** [comment]
Whoops didn't see that dupe, would be great if GitHub search properly worked

**Comment by Chew** [comment]
It do be like that honestly

**Comment by pop4959** [comment]
Yeah this is basically waiting on someone willing to redo the mail system and replace it with something more flexible.
